modules/edu.gemini.seqexec.model/src/test/scala/edu/gemini/seqexec/model/SharedModelArbitraries.scala

```diff
@@ -26,6 +26,12 @@ object SharedModelArbitraries {
     } yield SequencesQueue(Conditions.default, Some(Operator("operator")), b)
   }
 
+  implicit val clientIdArb: Arbitrary[ClientID] = Arbitrary {
+    for {
+      i <- Gen.uuid
+    } yield i
+  }
+
   implicit val instArb: Arbitrary[Instant] = Arbitrary {
```

Comment:
I think you could do the same thing here:
`implicit val instArb = Arbitrary(Gen.choose(0L, Long.MaxValue).map(Instant.ofEpochMilli(_)))`